src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -168,7 +168,7 @@ def load_arguments(self, _):
         c.argument('acr_use_identity', action='store_true', help="Enable or disable pull image from acr use managed identity")
         c.argument('acr_identity', help='Accept system or user assigned identity which will be set for acr image pull. '
                                         'Use \'[system]\' to refer system assigned identity, or a resource id to refer user assigned identity.')
-        c.argument('basic_auth', help='Enable or disable basic auth.', arg_type=get_enum_type(BASIC_AUTH_TYPES))
+        c.argument('basic_auth', help='Enable or disable basic auth for both SCM and FTP Basic Auth Publishing Credentials. Defaults to Enabled if not specified. Allowed values: Disabled, Enabled. See https://aka.ms/app-service-basic-auth to learn more.', arg_type=get_enum_type(BASIC_AUTH_TYPES))
```

Comment:
`arg_type=get_enum_type` will display the available values to users automatically.
Since this param has been defined with `arg_type=get_enum_type(BASIC_AUTH_TYPES)`, it is not necessary to specify the available values.
```suggestion
        c.argument('basic_auth', help='Enable or disable basic auth for both SCM and FTP Basic Auth Publishing Credentials. Defaults to Enabled if not specified. See https://aka.ms/app-service-basic-auth to learn more.', arg_type=get_enum_type(BASIC_AUTH_TYPES))
```